### PR TITLE
Update .NET documentation

### LINKED ISSUE #571: Update the .NET offering in README and website documentation
Two places to update:

1. one paragraph of README.md
image

2. one sentence in https://microsoft.github.io/FLAML/docs/Installation
image

### PR DESCRIPTION
**Pull request by luisquintanilla**
- [x] Update install instructions
- [x] Update README
- [x] Add tutorial

Fixes #571

### CODE REVIEW

**Review by sonichi — COMMENTED** [review]
Looks good to me. One suggestion: Make the section *Optional Python Dependencies* a subsection of *Python*.

Also, is there anything to update about the .NET part in the README.md file?

File: website/docs/Installation.md [hunk 1]
```diff
@@ -11,9 +13,22 @@ or conda:
...
+- *ML.NET CLI* - A dotnet CLI tool for training machine learning models using FLAML on Windows, MacOS, and Linux. For more information on how to install the ML.NET CLI, see the [install the ML.NET CLI](https://docs.microsoft.com/dotnet/machine-learning/how-to-guides/install-ml-net-cli?tabs=windows) guide. 
+
+**Code-first**
+
+- [*Microsoft.ML.AutoML*](https://www.nuget.org/packages/Microsoft.ML.AutoML/0.20.0-preview.22313.1) - NuGet package that provides direct access to the FLAML AutoML APIs that power low-code solutions like Model Builder and the ML.NET CLI. For more information on installing NuGet packages, see the install and use a NuGet package in [Visual Studio](https://docs.microsoft.com/nuget/quickstart/install-and-use-a-package-in-visual-studio) or [dotnet CLI](https://docs.microsoft.com/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli) guides.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply website/docs/Installation.md:29]: That makes sense and yes! I'll update the README as well.

**Comment by luisquintanilla** [comment]
@sonichi any thoughts on what might be causing the build issues? I've only changed the md files. I did pull the latest changes from `main`, so maybe there are conflicts there as well.

**Comment by sonichi** [comment]
> @sonichi any thoughts on what might be causing the build issues? I've only changed the md files. I did pull the latest changes from `main`, so maybe there are conflicts there as well.

@liususan091219 is working on fixing those issues in #647 . She has also mentioned it on [gitter](https://gitter.im/FLAMLer/community)

**Review by sonichi — APPROVED** [review]
Thanks. Looks good to me.

**Comment by luisquintanilla** [comment]
@sonichi this one's ready to merge but it looks like I need an additional approval.

**Pull request merged**